pkg/koordlet/resmanager/cgroup_reconcile.go

```diff
@@ -467,6 +467,8 @@ func makeCgroupResourcesForAnolis(owner *OwnerRef, parentDir string, summary *cg
 	var resources []MergeableResourceUpdater
 
 	if !system.HostSystemInfo.IsAnolisOS {
+		klog.V(6).Infof("ignored cgroup resources which required non Anolis OS, owner: %v, parentDir: %v",
```

Comment:
according to our convention, the highest level should be 5.

Reply:
done